src/states/game.js

```diff
@@ -24,9 +31,20 @@ Game.prototype =
     {
         ship_mass: 1,
         blackHole_mass: 500000,
-        camera_cutoff_x: 500
+        camera_cutoff_x: 500,
+        start_position: {
+            x: 100,
+            y: 300
+        },
+        score_constant: 0.05
     },
 
+    preload : function()
+    {
+        this.load.script('webfont',
```

Comment:
This would mean the game couldn't be played offline (or at least - the score couldn't be displayed offline), is there a way of bundling this with the game files? Is that something we're bothered about? I'm thinking if we want to create a phone app for this that doesn't require a data connection. Maybe this is something that we don't need to address for the prototype.

If we do go down this route, we should register loading this with the preload state, like we do for the images. This gives us quick loading times between states as it's all done during start up. It also means we're not trying to reload this script each time we go from gameover screen back to the main game.